src/components/event/details/DetailsSuspectedCommits.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <DetailsBase
+    class="details-suspected-commit"
+    :expand-showed="commits.length > numberOfVisibleCommits"
+    @expandClicked="isMoreCommitsShown = !isMoreCommitsShown"
+  >
+    <template #header>
+      {{ $t("event.suspectedCommits.header") }}
+    </template>
+    <template #content>
+      <div
+        v-for="(commit, index) in filteredCommits"
+        :key="index"
+        class="
+          event-details__content-block
+          details-suspected-commit__content-block
+        "
+        data-ripple
+      >
+        <div class="details-suspected-commit__left">
+          <div class="details-suspected-commit__left-title">
+            {{ commit.title }}
+          </div>
+          <div>
+            <EntityImage
+              :id="commit.author.charCodeAt(0).toString(16)"
```

Comment:
will the `author` always be defined? same question about `title`, `date` and `hash`